README.md

```diff
@@ -99,6 +99,12 @@ We are seeking for great minds to extend our core team. Contribution in any area
 
 **[Development environment setup](docs/developmentEnvironment.md)**
 
+## Publish NPM package
+
+When the abiniser directory changed:
+- bump the version property in the package.json
+- `npm publish --access public` 
```

Comment:
why do we need `--access public` ? isnt that the default unless set differently in package.json?

Reply:
The default for scoped packages, that they are published as private package. You need that --access public to publish as a public package (at least the documentation said that :)